gui/templates/base.html

```diff
@@ -102,34 +102,52 @@ <h1 style="word-wrap:break-word">{{ message.message }}</h1>
         {% endfor %}
       </div>
       {% endif %}
-      <div class="w3-container">
-        <h1><a href="/">My Lnd Overview</a></h1>
-      </div>
       {% block content %}{% endblock %}
     </div>
   </body>
-  <footer>
-    <div id="footer">
-      <div class="w3-container w3-padding-small">
-        <center><button id="toggleTheme" onclick="toggleTheme()">{% if request.COOKIES.darkmode == 'true' %}Light{%else%}Dark{% endif %} Theme</button></center>
-      </div>
-      <div class="w3-container w3-padding-small">
-        <center>LNDg v1.9.0</center>
-        <center><a href="{% url 'logout' %}">Logout</a></center>
-      </div>
+  <footer id="footer">
+    <div class="w3-container w3-padding-small" style="margin-bottom:50px">
+      <center>LNDg v1.8.0</center>
+    </div>
+    <div class="w3-bar w3-black" style="position:fixed;bottom:0px;width:100%;z-index:100">
+      <a class="w3-bar-item w3-hover-blue" style="padding:0" title="Dashboard" href="/"><img height="38px" src="/favicon.ico"></img></a>
+      <a class="w3-bar-item w3-hover-blue" href="/income">P&L</a>
+      <a class="w3-bar-item w3-hover-blue" href="/closures">Closures</a> 
+      <a class="w3-bar-item w3-hover-blue" href="/opens">New Peers</a>
+      <a class="w3-bar-item w3-hover-blue" href="/peerevents">Peer Events</a>
+      <a class="w3-bar-item w3-hover-blue" href="/actions">AR Actions</a> 
+      <a class="w3-bar-item w3-hover-blue" href="/fees">Fee Rates</a>
+      <a class="w3-bar-item w3-hover-blue" href="/autopilot">Autopilot</a>
+      <a class="w3-bar-item w3-hover-blue" href="/autofees">Fee Log</a>
+      <a class="w3-bar-item w3-hover-blue" href="/channels">Channel Performance</a>
+      <a class="w3-bar-item w3-hover-blue" href="/keysends">Keysends</a>
+      <a class="w3-bar-item w3-hover-blue" href="/rebalancing">Rebalancing</a>
+      <a class="w3-bar-item w3-hover-blue" href="/towers">Towers</a>
+      <a class="w3-bar-item w3-hover-blue" href="/batch">Batching</a>
+      <a class="w3-bar-item w3-hover-blue" href="/trades">Trades</a>
+      <a class="w3-bar-item w3-hover-blue" href="/advanced">Advanced Settings</a>
+      <a class="w3-bar-item w3-hover-blue" href="/logs/?tail=20">Logs</a>
+      <span class="w3-padding-small w3-right" style="max-height:30px;">
+        <a class="w3-bar-item w3-hover w3-text-white" title="Last Update" id="refresh_stats" style="padding:8px 0px"></a>
+        <a class="w3-bar-item w3-hover w3-btn" title="Toggle Theme" id="themeToggler" style="padding:8px">{% if request.COOKIES.darkmode == 'true' %}☀️{%else%}🌙{% endif %}</a>
+        <a class="w3-bar-item w3-hover w3-text-red" title="Logout" href="{% url 'logout' %}" style="padding:8px">⏻</a>
```

Comment:
The `logout` button is not rendering like your screenshots, not sure if it made it after the git push.

![image](https://github.com/cryptosharks131/lndg/assets/38626122/0e992d7d-c319-48ad-bd3a-3bfe6a991329)

I also noticed we have an issue with the logout function in Django >= 5.0 as it no longer allows a GET request to the `logout` URL and must use POST.